api/src/main/java/javax/config/ConfigAccessor.java

```diff
@@ -96,6 +96,17 @@
      */
     ConfigAccessor<T> useConverter(Converter<T> converter);
 
+    /**
+     * Considers the type of the configuration entry a composed type.
+     * This method enables developers to define the look-up of composed type in scenarios where they cannot annotate
+     * the composed type with {@link ComposedConfig}. This is the programmatic equivalent of injecting a composed type
+     * with qualifier {@link ComposedConfig} at the injection point.
+     *
+     * @return This builder
+     * @see ComposedConfig
+     */
+    ConfigAccessor<T> asComposed();
```

Comment:
hmm, if you pass in an interface, then it might treat it as composed automatically?

Reply:
Yes, if we require that the type is always annotated with `@ComposedConfig` we don't need that.